containers/test-apps/jwks-server/resources/jwks.json

```diff
@@ -0,0 +1,90 @@
+{
```

Comment:
We should generate these keys dynamically as part of the startup script.

Reply:
1. We don't have to answer the question of how these keys are generated.
2. We ensure there's nothing special about the values in this file.

Reply:
Since it is not particularly easy to generate these values with our current library, I'd vote to keep it as it is. This is a simple test server (or file used by unit tests in another context) implementation, I'd like to keep it simple and avoid additional features like generating these keys dynamically.

Reply:
OK.  It would be helpful for you to document how you generated the keys.  This server must remain Travis-only for as long as private keys are in the repository.

Reply:
Added sources of the stored keys in the README.